### PR TITLE
add: product-page && card-component

### PR DESCRIPTION
**Pull request by amantulsyan35**

### CODE REVIEW

File: src/Pages/Products/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+const ProductFilter = ({ label, type }) => {
+  return (
+    <div className='prod-func'>
+      <input type={type} />
+      <label>{label}</label>
+    </div>
+  );
+};
+
```
Comment (bhtibrewal) [inline src/Pages/Products/index.js:19]: suggestion: move these components to a different file

**Review by bhtibrewal — APPROVED** [review]
just 1 or 2 suggestions, everything else looks good, ok to merge

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after @bhtibrewal suggestion. Nice work!

File: src/App.js [hunk 2]
```diff
@@ -4,13 +4,15 @@ import { Routes, Route } from 'react-router-dom';
 
 import Navbar from './components/Navbar/index';
 import Homepage from './Pages/Homepage';
+import Product from './Pages/Products';
```
[the remark this answers is not in the document]
Comment (amantulsyan35) [reply src/App.js:7]: That's a good suggestion, thanks

File: src/Pages/Products/index.js (hunk 1 above)
Comment (amantulsyan35) [reply src/Pages/Products/index.js:19]: These files will not be used anywhere, that's why they are here

**Pull request closed without merging**